tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -342,6 +321,105 @@ def test_multiple_fsx(
     run_benchmarks(remote_command_executor, scheduler_commands)
 
 
+@pytest.mark.usefixtures("instance", "scheduler")
+def test_multi_az_fsx(
+    os,
+    region,
+    fsx_factory,
+    svm_factory,
+    open_zfs_volume_factory,
+    vpc_stack,
+    pcluster_config_reader,
+    clusters_factory,
+    s3_bucket_factory,
+    scheduler_commands_factory,
+    test_datadir,
+):
+    bucket_name = s3_bucket_factory()
+    bucket = boto3.resource("s3", region_name=region).Bucket(bucket_name)
+    bucket.upload_file(str(test_datadir / "s3_test_file"), "s3_test_file")
+    import_path = "s3://{0}".format(bucket_name)
+    export_path = "s3://{0}/export_dir".format(bucket_name)
+
+    partition = utils.get_arn_partition(region)
+    num_existing_fsx_ontap_volumes = 2 if partition != "aws-cn" else 0  # China does not have Ontap
+    num_existing_fsx_open_zfs_volumes = 2 if partition == "aws" else 0  # China and GovCloud do not have OpenZFS.
+    num_existing_fsx_lustre = 2
+
+    fsx_lustre_mount_dirs = [f"/fsx_lustre_mount_dir{i}" for i in range(num_existing_fsx_lustre)]
+    fsx_ontap_mount_dirs = [f"/fsx_ontap_mount_dir{i}" for i in range(num_existing_fsx_ontap_volumes)]
+    fsx_open_zfs_mount_dirs = [f"/fsx_open_zfs_mount_dir{i}" for i in range(num_existing_fsx_open_zfs_volumes)]
+
+    existing_fsx_lustre_fs_ids = _create_fsx_lustre_volume_ids(
+        num_existing_fsx_lustre, fsx_factory, import_path, export_path
+    )
+    fsx_on_tap_volume_ids = _create_fsx_on_tap_volume_ids(num_existing_fsx_ontap_volumes, fsx_factory, svm_factory)
+    fsx_open_zfs_volume_ids = _create_fsx_open_zfs_volume_ids(
+        num_existing_fsx_ontap_volumes, fsx_factory, open_zfs_volume_factory
+    )
+
+    cluster_config = pcluster_config_reader(
+        config_file="pcluster-managed-fsx.config.yaml",
```

Comment:
is this correct? I think it should point to `pcluster.config.yaml`

Reply:
Good catch. Left over from a local test. Fixing this ...

Reply:
Since the test here is checking different behavior between managed and unmanaged fsx renaming this config into pcluster-unmanaged-fsx.config.yaml (and accordingly the local variable) would make the code more readable